cmd/der2ascii/main.go

```diff
@@ -23,6 +24,7 @@ import (
 
 var inPath = flag.String("i", "", "input file to use (defaults to stdin)")
 var outPath = flag.String("o", "", "output file to use (defaults to stdout)")
+var isPem = flag.Bool("p", false, "treat the input as a PEM file")
```

Comment:
WDYT about `-pem` instead of `-p`? I suppose this is my fault for using `-i` and `-o` instead of `-in` and `-out`, but I guess those are a bit more common across tools, whereas PEM is a bit more niche.